jekyll/_cci2/audit-logs.adoc

```diff
@@ -63,4 +63,35 @@ The following are the system events that are logged. See `action` in the Field s
 - **version:** Version of the event schema. Currently the value will always be 1. Later versions may have different values to accommodate schema changes.
 - **scope:** If the target is owned by an Account in the CircleCI domain model, the account field should be filled in with the Account name and ID. This data is a JSON blob that will always contain `id` and `type` and will likely contain `name`.
 - **success:** A flag to indicate if the action was successful.
-- **request:** If this event was triggered by an external request this data will be populated and may be used to connect events that originate from the same external request. The format is a JSON blob containing `id` (the unique ID assigned to this request by CircleCI).
\ No newline at end of file
+- **request:** If this event was triggered by an external request this data will be populated and may be used to connect events that originate from the same external request. The format is a JSON blob containing `id` (the unique ID assigned to this request by CircleCI).
+
+== Request audit logs from the web app
+
+CircleCI server customers, as well as performance and scale cloud customers, can request audit logs from the web app. To have access to this feature, cloud customers must be organization admins.
+
+Navigagte to **Organization Settings > Audit Logs**. Select a date range from the dropdown, then click the **Request audit logs** button. The earliest start date is one year ago from the current day, and the lastest end date is the current day. The data available will depend on the data retention period set per organization, so the returned time period for the data could be less than the requested timeframe.
+
+NOTE: Submitted dates are in UTC. To avoid issues arising from potential time zone differences, CircleCI adds extra time to the request. For example, if you request August 2 - August 5, the returned results will be from Aug 1 - Aug 6. The audit log request column is also displayed in UTC.
```

Comment:
```suggestion
NOTE: Submitted dates are in UTC. To avoid issues arising from potential time zone differences, CircleCI adds extra time to the request. For example, if you request August 2 - August 5, the returned results will be in the range August 1 - August 6. The audit log request column is also displayed in UTC.
```